src/pages/api/breadcrumbs.ts

```diff
@@ -0,0 +1,72 @@
+import apiUrl from '../../utils/apiUrl';
+import { ApiFetch, createApiFetch, RequestWithHeaders } from '../../utils/apiFetch';
+import { NextApiRequest, NextApiResponse } from 'next';
+
+export default async (
+    req: NextApiRequest,
+    res: NextApiResponse,
+): Promise<void> => {
+    const orgId = req.query.orgId;
+
+    if (!orgId) {
+        return res.status(400).json({ error: 'orgId not provided' });
+    }
+
+    const apiFetch = createApiFetch(req as RequestWithHeaders);
+
+    const { query } = req;
+    const pathname  = query.pathname as string;
```

Comment:
I would move these to the top of the function, and use destructoring to retrieve `pathname` and `orgId` in a single line.